.changeset/chilled-bottles-hug.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react-brand': patch
+---
+
+Minor internal refactoring to `VideoPlayer` component
```

Comment:
Could there be a scenario where someone was passing a ref as a prop, which now won't work because we're no longer spreading it in?